src/parser/jobs/pld/modules/Weaving.tsx

```diff
@@ -0,0 +1,43 @@
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import CoreWeaving from 'parser/core/modules/Weaving'
+
+const SPELLS = [
+	ACTIONS.HOLY_SPIRIT.id,
+	ACTIONS.HOLY_CIRCLE.id,
+	ACTIONS.CONFITEOR.id,
+	ACTIONS.CLEMENCY.id,
+]
+
+export default class Weaving extends CoreWeaving {
+
+	private isRequiescatActive: boolean = false
```

Comment:
To avoid redoing the work of tracking the buff window, you can replace this and the constructor hooks with the `combatants` dependency, and then change the `this.isRequiescatActive` check in the function below to `this.combatants.selected.hasStatus(STATUSES.REQUIESCAT.id)`. See [SAM's Combos.ts](https://github.com/xivanalysis/xivanalysis/blob/master/src/parser/jobs/sam/modules/Combos.ts) for a Typescript example.

Reply:
Should probably update this in a few other places, but good to know for this review.